ignite/templates/app/files/cmd/{{binaryNamePrefix}}d/cmd/commands.go.plush

```diff
@@ -124,7 +133,66 @@ func newApp(
 	traceStore io.Writer,
 	appOpts servertypes.AppOptions,
 ) servertypes.Application {
-	baseappOptions := server.DefaultBaseappOptions(appOpts)
+	var cache storetypes.MultiStorePersistentCache
```

Comment:
nit, if you nit an extra baseapp option, let's append it here.
Which one have you added compared to DefaultBaseappOptions?

Reply:
Because before I did `in-place` on a chain using sdk47 there was a difference that led to an error in `baseappOptions` so I rewrote it. For the chain from ignite I don't see that happening, sorry for this.